app/presenters/tree_node/menu/item.rb

```diff
@@ -0,0 +1,36 @@
+module TreeNode
+  module Menu
+    class Item < TreeNode::Menu::Node
+      set_attribute(:key)      { "#{@options[:node_id_prefix]}__#{@object.feature}" }
+      set_attribute(:title)    { _(details[:name]) }
+      set_attribute(:tooltip)  { _(details[:description]) || _(details[:name]) }
+      set_attribute(:selected) { self_selected? || select_by_children_selected }
+
+      def children
```

Comment:
...but.. how? =-O Sorcery :)

In my book, running `TreeNode...new` for each child and returning the resulting array does exactly what I said..

.. but @skateman enlightened me that `children` method does not behave the same as `:chlildren` field on hash nodes, so .. sorry about that :)

But .. yeah, you are buiding the node for each child regardless, you're just not using it after :).

Anyway, what he said :)